src/map/glmap/interaction.actions.js

```diff
@@ -65,9 +67,20 @@ export const mapHover = (latitude, longitude, features, cluster) => (dispatch, g
   if (isEmpty === true) {
     const feature = findFeature(features, null)
     if (feature !== undefined) {
+      event.type = 'static'
+      event.cluster = cluster
+      event.layer = {
+        id: feature.staticLayerId,
+      }
+      const properties = feature.feature.properties
+      event.target = {
+        properties,
+      }
+      cursor = 'pointer'
```

Comment:
Reminder for the future: send an array of features that intersects on this click event.